pvlib/irradiance.py

```diff
@@ -2819,3 +2820,130 @@ def dni(ghi, dhi, zenith, clearsky_dni=None, clearsky_tolerance=1.1,
             (zenith < zenith_threshold_for_zero_dni) &
             (dni > max_dni)] = max_dni
     return dni
+
+
+def pvfactors_bifacial_irradiance(
```

Comment:
Talking with @wholmgren we would like this function in a new module `pvlib.bifacial`. That creates space for other bifacial contributions, and helps users discover this function.  We might put a wrapper function in `pvlib.irradiance`, `get_bifacial(xxxx, model='pvfactors')` or something like that.

Function name: I think you can drop `bifacial` since the function will be in `pvlib.bifacial`

Reply:
Thank you @cwhanse ! That makes a lot of sense. I moved everything to new bifacial modules here: 1c976e3. 
I also reverted my ``autopep8`` changes for a couple of modules in a340c20.
I'll be working on the ``pandas`` IO handling shortly! 
I'm currently a bit confused about why the ``Appveyor`` tests fail even though ``Travis`` passes, so I'll keep looking

Reply:
Oh I see, Travis actually skipped the bifacial tests

Reply:
You'll need to add the required packages to the ci files in https://github.com/pvlib/pvlib-python/tree/master/ci and https://github.com/pvlib/pvlib-python/blob/master/appveyor.yml

Reply:
Thanks! I also just realized that ``pvlib``'s testing relies on ``conda`` and not ``pip``. I'll try to learn it and spend some time on creating conda builds for ``pvfactors``. My first attempt was a failure because the pvlib conda channel has pvlib versions up to ``0.6.0``, and I had ``>=0.6.1b0`` as a requirement in pvfactors, so it seems like it will take more time than I thought

Reply:
`pvlib`'s CI configurations use `conda` for most packages for convenience, but the tests don't require it. There's no need to make conda builds for `pvfactors` for this purpose. You can specify `pvfactors` in the `pip` section at the bottom of the requirements files. Our appveyor tests only use conda, but we can skip the pvfactors tests on that platform.

Reply:
Ok, thanks a lot for all the help @wholmgren